core/src/main/java/org/elasticsearch/action/delete/DeleteResponse.java

```diff
@@ -94,7 +89,8 @@ public String toString() {
         builder.append(",type=").append(getType());
         builder.append(",id=").append(getId());
         builder.append(",version=").append(getVersion());
-        builder.append(",found=").append(found);
+        builder.append(",found=").append(isFound());
+        builder.append(",operation=").append(getOperation().getLowercase());
```

Comment:
If you always want to use the lowercased version of the constant maybe just implement `toString` to return lowercased? I'm not sure what the right thing is here but it isn't a big deal either way.